### ISSUE TITLE
SVFAP pre-training

### ISSUE THREAD

**Issue by ztt159** [sunlicai/SVFAP#1]
Hello, concerning the SVFAP code that is forthcoming for open-source release, could you please advise if it is feasible to utilize it as a pre-trained model for the purpose of feature extraction on novel datasets, akin to what is achievable with a Masked Autoencoder (MAE)?

**Comment by sunlicai** [comment]
Hi @ztt159, thanks for your interest in our work!

Definitely yes! You can use the pre-trained model for feature extraction and also can fine-tune it for your datasests.

We plan to release the code in this month. Stay tuned!

**Comment by Utkarsh-shift** [comment]
Please provide code

**Comment by sunlicai** [comment]
Hi @ztt159 and @Utkarsh-shift,  I am very sorry for the late reply. The code and models are available now.
